src/extension.ts

```diff
@@ -1789,14 +1789,24 @@ export class Ext extends Ecs.System<ExtEvent> {
             this.register({ tag: 3, window });
         });
 
-        this.connect(display, 'grab-op-begin', (_, _display, win, op) => {
-            this.on_grab_start(win, op);
-        });
+        if (GNOME_VERSION?.startsWith("3.")) {
+            // GNOME 40 removed the first argument of the callback
```

Comment:
this comment should go to else statement

Reply:
Done